lib/instrumental_tools/system_inspector.rb

```diff
@@ -0,0 +1,67 @@
+class SystemInspector
+  TYPES = [:gauges, :incrementors]
+  attr_accessor *TYPES
+
+  def self.memory
+    @memory ||= Memory.new
+  end
+
+  def initialize
+    @gauges = {}
+    @incrementors = {}
+    @platform =
+      case RUBY_PLATFORM
+      when /linux/
+        require "instrumental_tools/system_inspector/linux"
+        SystemInspector::Linux
+      when /darwin/
+        require "instrumental_tools/system_inspector/osx"
+        SystemInspector::OSX
+      else
+        raise "unsupported OS"
+      end
+  end
+
+  def self.command_missing(command, section)
```

Comment:
Yeah, I misread this entirely. I was thinking it was a boolean check because <reason I no longer remember>